test/functional/inspec_exec_junit_test.rb

```diff
@@ -11,55 +11,55 @@
     out = inspec('exec ' + example_control + ' --format junit --no-create-lockfile')
     out.stderr.must_equal ''
     out.exit_status.must_equal 0
-    doc = Nokogiri::XML(out.stdout)
-    doc.errors.length.must_equal 0
+    doc = REXML::Document.new(out.stdout)
+    doc.has_elements?.must_equal true
```

Comment:
Isn't that a different test? before we checked the errors in doc but now we just check that doc has elements?